airbyte-integrations/connectors/source-amazon-ads/integration_tests/configured_catalog_report.json

```diff
@@ -14,6 +14,21 @@
       },
       "sync_mode": "incremental",
       "destination_sync_mode": "overwrite"
+    },
+    {
+        "stream": {
+        "name": "sponsored_brands_v3_report_stream",
+        "json_schema": {},
+        "supported_sync_modes": ["full_refresh", "incremental"],
+        "source_defined_primary_key": [
+          ["profileId"],
+          ["recordType"],
+          ["reportDate"],
+          ["recordId"]
+        ]
+      },
```

Comment:
please add `default_cursor_field`, `cursor_field`, and `primary_key`

Reply:
Added cursor_field; other fields are defined in streams.